apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -89,11 +84,6 @@ class SectionProgress extends Component {
   onChangeLevel = lessonOfInterest => {
     this.props.setLessonOfInterest(lessonOfInterest);
 
-    this.recordEvent('jump_to_lesson', {
-      script_id: this.props.scriptId,
-      stage_id: this.props.scriptData.lessons[lessonOfInterest].id,
-    });
-
     analyticsReporter.sendEvent(EVENTS.PROGRESS_JUMP_TO_LESSON, {
       sectionId: this.props.sectionId,
       unitId: this.props.scriptId,
```

Comment:
I think line 105 and the `recordEvent` function can be removed here too?

Reply:
Ideally yes I will in the future! The only issue here is that there's not yet a corresponding Amplitude event for that line:/